### PR TITLE
Support for mount_options for StorageClass

### PR DESCRIPTION
**Pull request by cwebbtw**
Resolved https://github.com/terraform-providers/terraform-provider-kubernetes/issues/510

See also:

https://github.com/terraform-providers/terraform-provider-kubernetes/pull/619

- Implementation *slightly* differs
- Added acceptance tests
- Added documentation

Happy to merge with pull request 619, or be considered separately.

### CODE REVIEW

**Comment by cwebbtw** [comment]
@alexsomesan hey! Would it be possible to get this reviewed as we're currently using a horrible workaround and would really like to continue using this provider. Thank you! 👍

File: kubernetes/resource_kubernetes_storage_class.go [hunk 1]
```diff
@@ -108,6 +120,7 @@ func resourceKubernetesStorageClassRead(d *schema.ResourceData, meta interface{}
 	d.Set("storage_provisioner", storageClass.Provisioner)
 	d.Set("reclaim_policy", storageClass.ReclaimPolicy)
 	d.Set("volume_binding_mode", storageClass.VolumeBindingMode)
+	d.Set("mount_options", storageClass.MountOptions)
```
Comment (alexsomesan) [inline kubernetes/resource_kubernetes_storage_class.go]:
```suggestion
	d.Set("mount_options", newStringSet(schema.HashString, storageClass.MountOptions))
```

**Comment by cwebbtw** [comment]
> Please see the spot comment about creating the schema.Set value.

Thank you @alexsomesan for suggesting the change, it has been accepted. The CI tests seem to be passing, so just waiting for GitHub to recognise that the travis run has passed.

**Comment by mjromper** [comment]
Any idea when this PR is getting merged? It's quite an important feature.

**Comment by cwebbtw** [comment]
Nothing left to do from me AFAIK. Comments have been actioned.

@alexsomesan any ideas when this can get merged? :-)

**Comment by phalcon30964** [comment]
Hello, any idea when this can be merged?, we need this too

**Pull request merged**